core/adapters/json_parse.go

```diff
@@ -52,26 +52,27 @@ func (jpa *JSONParse) Perform(input models.RunInput, _ *store.Store) models.RunO
 		return models.NewRunOutputError(err)
 	}
 
-	js, err := simplejson.NewJson([]byte(val))
```

Comment:
`input.Result()` is of type gjson.Result already. I made the assumption that the input was not previously checked to be correctly encoded as JSON.  I can remove this line if that is not the case.

Reply:
Seems like the safe option for now.